### ISSUE TITLE
The loss label of of NCESoftmaxLoss in NCECriterion.py?

### ISSUE THREAD

**Issue by xuChenSJTU** [HobbitLong/CMC#46]
Hi,

I see your code for NCESoftmaxLoss as follows:

#########
class NCESoftmaxLoss(nn.Module):
    """Softmax cross-entropy loss (a.k.a., info-NCE loss in CPC paper)"""
    def __init__(self):
        super(NCESoftmaxLoss, self).__init__()
        self.criterion = nn.CrossEntropyLoss()

    def forward(self, x):
        bsz = x.shape[0]
        x = x.squeeze()
        label = torch.zeros([bsz]).cuda().long()
        loss = self.criterion(x, label)
        return loss
###########
The label for this loss is label = torch.zeros([bsz]).cuda().long(), but in your paper, according to eq.2, 
![image](https://user-images.githubusercontent.com/24096174/75645447-082b5d00-5c9a-11ea-9480-0b1a78192a42.png)
You have one positive for each sample. 

So is something missed here??

Thanks.
